stratum/hal/bin/bcm/standalone/docker/Dockerfile

```diff
@@ -11,4 +11,4 @@ RUN rm ./stratum_bcm_deb.deb
 
 EXPOSE 28000/tcp
 EXPOSE 9339/tcp
-ENTRYPOINT start-stratum.sh
+ENTRYPOINT ["/usr/bin/start-stratum.sh"]
```

Comment:
We should try the relative paths first, before hardcoding them. Otherwise we have the same information in two places (here and the BUILD file)

Reply:
make sense, since we placed the script in the default system binary path.